cookiecutter-xblock/{{cookiecutter.repo_name}}/.github/workflows/ci.yml

```diff
@@ -48,4 +41,4 @@ jobs:
       uses: codecov/codecov-action@v3
       with:
         flags: unittests
-        fail_ci_if_error: true
+        fail_ci_if_error: true
```

Comment:
We use .editorconfig files to make editor settings the same among collaborators. If you enable editorconfig support in your editor or IDE, these kinds of accidental changes won't happen. https://editorconfig.org/